code/ship/awacs.cpp

```diff
@@ -221,6 +216,19 @@ float awacs_get_level(object *target, ship *viewer, int use_awacs)
 
 		check_huge_ship = (sip->is_huge_ship());
 	}
+
+	// check for an exempt ship. Exempt ships are _always_ visible
+	if (target->type == OBJ_SHIP) {
+		Assert(shipp != NULL);
```

Comment:
Since you're going to add an assert check against NULL here, modernize the function with `nullptr`.  So initialize the variable with `nullptr` intead of `NULL` and replace `NULL` in all the places it is checked.  

Basically, it's not worth it to change all the `NULL`s in the codebase to `nullptr` at once since the benefit is small, but we shouldn't add new `NULL`s  in PRs and should update old code that is in the same function.  Might be a good time to update `ship_info *sip = NULL;`, too.

Reply:
Done